src/components/ProductListPage/ProductListPage.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+
+import Filter from './Filters';
+import ProductList from '../ProductList';
+import Heading from '../Heading/Heading';
+
+import './ProductListPage.scss';
+
+export const CN = 'productListPage';
+
+class ProductListPage extends Component {
+  render() {
+    // console.log(this.props);
+    const { match: { params: { category } } } = this.props;
+
+    return (
+      <div className={`content ${cx(CN)}`}>
+        <div className={`${CN}__line`}>
+          <div> Home</div>
+          <div>&#10094; Return to previous page</div>
+        </div>
+        <div className={`${cx(CN)}__gridWrapper`}>
+          <div className={`${CN}__filter`}>
+            <Filter category={category} />
+            {this.category}
+          </div>
+          <div className={`${CN}__poster`}>
+            <Heading className={`${CN}__title`} title={category} />
+            <div
+              className={`${CN}__poster-wrapper container`}
+              style={{
+                backgroundImage: `url(src/assets/img/productListPage/${category}.jpg)`
+              }}
+            />
+          </div>
+          <div className={`${CN}__productList`}>
+            <ProductList />
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default ProductListPage;
```

Comment:
2 complaints:
______________
1. rewritte to functional component
2. add propTypes validation
______________
Explanation: If all that your component does is rendering UI - than use functional component.

Reply:
Sure, I will rewrite it